### PR TITLE
React 16 update

### PR DESCRIPTION
**Pull request by arijitdasgupta**

### CODE REVIEW

File: src/TestCaseFactory.js [hunk 1]
```diff
@@ -108,11 +113,17 @@ function createFromClass(reactClass, props = {}) {
...
-    render: function render() {
+  class StatelessComponent extends React.Component {
+    render() {
       return statelessFunctionalComponent(this.props);
-    },
-  });
+    }
+  }
+  return StatelessComponent;
+  // return React.createClass({
```
Comment (highjacker) [inline src/TestCaseFactory.js]: remove unused code

File: src/TestCaseFactory.js [hunk 2]
```diff
@@ -78,11 +78,16 @@ function createFromElement(reactElement) {
...
-  const Wrapper = React.createClass({
-    render: function() {
+  class Wrapper extends React.Component {
+    render() {
       return React.createElement('div', {children: this.props.children});
-    },
-  });
+    }
+  }
+  // const Wrapper = React.createClass({
```
Comment (highjacker) [inline src/TestCaseFactory.js]: remove unused code

Comment (arijitdasgupta) [reply src/TestCaseFactory.js]: Done

File: src/TestCaseFactory.js (hunk 1 above)
Comment (arijitdasgupta) [reply src/TestCaseFactory.js]: Done

**Comment by arijitdasgupta** [comment]
PRA @highjacker

**Pull request merged**